src/containers/Application.js

```diff
@@ -297,8 +299,14 @@ it is accessible by the sidebar toggler button.
 		if (this.state.SplashscreenRequestors > 0) return (
 			<Provider store={this.Store}>
 				<div className="app">
-					<AlertsComponent app={this}/>
-					<SplashScreen app={this}/>
+					<AlertsComponent app={this} />
+					<SplashScreen app={this} />
+					{this.Config.get('site_title') != null && this.Config.get('site_title') != undefined ?
+						<Helmet>
+							<title>{this.Config.get('title') ? this.Config.get('site_title') + " | " + this.Config.get('title') : this.Config.get('site_title')}</title>
+						</Helmet>
+						: { null }
```

Comment:
@wuujcik Not sure about the `null` wrapped in curly brackets here and on line 325 https://github.com/TeskaLabs/asab-webui/pull/65/files#diff-31d3784ef11656fbccbce9b6b554c572b43dfad75c897ef7fa510ba7a443735fR325. I would suggest to remove the `null` brackets, otherwise it can cause crash of the app.